android/build.gradle

```diff
@@ -1,27 +1,10 @@
-
```

Comment:
Probably the best practice would be to include a buildscript block in it with the repository and Android plugin Gradle version. That way it stands alone better, and it's not implicitly dependent on being in a project that has a top-level build.gradle file.

Don't worry because if you include this module, with its own buildscript block, in a project that has a top-level build.gradle with a competing buildscript block, the top-level build file will win and it will ignore the block in your module. 

As a standalone library, we define all dependencies and its version to make sure everything run correctly with the the tested configuration.